content/13.conclusion.md

```diff
@@ -1 +1,7 @@
 ## Conclusion {#conclusion}
+
+Deep learning techniques have the potential for wide use in biology, meeting or exceeding the performance of both humans and the current state-of-the art algorithms in a variety of tasks.
+Beyond simply achieving performance, deep learning has the potential to generate novel biological insights that may then be applied back to fundamental research.
+To realize this potential, the use of deep learning as a research tool must be approached as any other tool would be: scientifically and thoughtfully.
+We hope that the tips we present here will serve as a starting point for the discussion of best practices for deep learning as they apply to biology, not as an ending point.
+As the field of deep learning matures, with its theoretical underpinnings more fully understood, the best practices for its use will surely evolve as well.
```

Comment:
Again, nothing wrong with it, but very speculative. I am not sure if we need that sentence.